pandas/core/frame.py

```diff
@@ -115,6 +115,13 @@
     side, respectively
 copy : boolean, default True
     If False, do not copy data unnecessarily
+indicator : boolean, default False
+    If True, adds a column to output DataFrame called "_merge" with 
+    information on the source of each row. "_merge" is Categorical-type 
+    and takes on a value of "left_only" for observations whose merge key 
```

Comment:
I don't think it has this functionality:

http://stackoverflow.com/questions/30358401/is-there-a-way-to-create-statas-merge-indicator-variable-with-rs-merge